### PR TITLE
Updates Hello World docs to note svc DNS & endpoints parameter

### LINKED ISSUE #292: Hello-world application docs needs a few updates
These are mostly nits, but as I ran through my setup, I found a couple issues with docs (happy to submit a PR for them) for the hello world app.

https://github.com/gluster/gluster-kubernetes/blob/master/docs/examples/hello_world/README.md#create-a-storageclass-for-our-glusterfs-dynamic-provisioner

1. In the storageclass yaml, there's a space in front of ` parameters:` which causes kube to reject it if you copy-paste it.

2. In the storageclass yaml, at least under Kube 1.7, kube rejects adding the storageclass with the `endpoint` parameter. 

3. It may be worth pointing out that for the `resturl` you should be able to use the kube DNS entry for the heketi svc.

Regardless, hello-world application document was helpful in validating my setup. Appreciated the document quite a bit, well done.

### PR DESCRIPTION
**Pull request by dougbtv**
Fixes #292

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/322)

### CODE REVIEW

File: docs/examples/hello_world/README.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ parameters:
 
 <2> Provisioner
 
-<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints)
+<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints). As of Kubernetes 1.7, this parameter should be removed as Kubernetes will reject this YAML definition.
 
-<4> Heketi REST Url, taken from Step 1 above (echo $HEKETI_CLI_SERVER)
+<4> Heketi REST Url, taken from Step 1 above (echo $HEKETI_CLI_SERVER), may also the Kubernetes service DNS name for the Heketi service.
```
Comment (jarrpa) [inline docs/examples/hello_world/README.md]: +`be` in `may also the` :)

File: docs/examples/hello_world/README.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ parameters:
 
 <2> Provisioner
 
-<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints)
+<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints). As of Kubernetes 1.7, this parameter should be removed as Kubernetes will reject this YAML definition.
```
Comment (humblec) [inline docs/examples/hello_world/README.md]: Even in 1.6, we dont need the endpoint in storageclass. @jarrpa

Comment (jarrpa) [inline docs/examples/hello_world/README.md]: Oh, right, yes. Should we just remove the endpoint parameter form the demo in general?

Comment (dougbtv) [inline docs/examples/hello_world/README.md]: I updated it to `>= 1.6` -- thinking maybe it's appropriate as there's instructions relative to kube 1.5 above? Assuming that parameter would be necessary with Kube 1.5, worth keeping if there's 1.5 instructions? *shrug*

File: docs/examples/hello_world/README.md (hunk 1 above)
Comment (dougbtv) [inline docs/examples/hello_world/README.md]: Thank you for the eyes! ...Cleaned that up to actually `reed liek englush`.

**Comment by dougbtv** [comment]
Tweaks in per the comments (didn't remove the parameter though, figure it's worthwhile to note as slong as there's other Kube 1.5 instructions in there), thanks guys, appreciate it.

**Comment by jarrpa** [comment]
Okay, fair. LGTM!

**Pull request merged**